src/OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -95,6 +96,34 @@ Future<ErrorMessageOr<CaptureListener::CaptureOutcome>> CaptureClient::Capture(
   return capture_result;
 }
 
+[[nodiscard]] static std::vector<ApiFunction> FindApiFunctions(
+    const orbit_client_data::ModuleManager& module_manager) {
+  // Api functions are declared in Orbit.h. They are implemented in user code through the
```

Comment:
I would move this comment outside of the block (above the function) for better visibility.

Reply:
Done.